diff --git a/node_modules/@umijs/request-record/dist/cjs/runtime/startMock.js b/node_modules/@umijs/request-record/dist/cjs/runtime/startMock.js
index 3465336..9e98b3d 100644
--- a/node_modules/@umijs/request-record/dist/cjs/runtime/startMock.js
+++ b/node_modules/@umijs/request-record/dist/cjs/runtime/startMock.js
@@ -32,7 +32,7 @@ var server = app.listen(port, () => {
     port
   );
 });
-app.get("*", (req, res) => {
+app.get("/{*path}", (req, res) => {
   const { url } = req;
   const key = `GET ${parse(url).pathname}`;
   if (mockFile[key]) {
@@ -41,7 +41,7 @@ app.get("*", (req, res) => {
     res.status(404).send(`Mock key ${key} Not Found`);
   }
 });
-app.post("*", (req, res) => {
+app.post("/{*path}", (req, res) => {
   const { url } = req;
   const key = `POST ${parse(url).pathname}`;
   if (mockFile[key]) {
diff --git a/node_modules/@umijs/request-record/src/runtime/startMock.js b/node_modules/@umijs/request-record/src/runtime/startMock.js
index 872f2f5..319890e 100644
--- a/node_modules/@umijs/request-record/src/runtime/startMock.js
+++ b/node_modules/@umijs/request-record/src/runtime/startMock.js
@@ -31,7 +31,7 @@ var server = app.listen(port, () => {
     port
   );
 });
-app.get('*', (req, res) => {
+app.get('/{*path}', (req, res) => {
   const { url } = req;
   const key = `GET ${parse(url).pathname}`;
   if (mockFile[key]) {
@@ -40,7 +40,7 @@ app.get('*', (req, res) => {
     res.status(404).send(`Mock key ${key} Not Found`);
   }
 });
-app.post('*', (req, res) => {
+app.post('/{*path}', (req, res) => {
   const { url } = req;
   const key = `POST ${parse(url).pathname}`;
   if (mockFile[key]) {